x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -624,6 +624,12 @@ export const InnerWorkspacePanel = React.memo(function InnerWorkspacePanel({
   );
 });
 
+function dispatchRenderComplete(node: HTMLDivElement | null) {
```

Comment:
There's also a `RenderCompleteDispatcher` class, but for just few lines I thought to inline it here

Reply:
cc @vadimkibana